R/load_files.R

```diff
@@ -1428,12 +1423,35 @@ get_EnVision_properties <- function(results.list, fname) {
 #' @return charvec with plate dims
 #' 
 .get_plate_size <- function(df) {
```

Comment:
consider removing this function (.get_plate_size is not exported function that was added during the recent refactor).